tests/Unit/Controllers/Service/Admin/CentreUserControllerTest.php

```diff
@@ -35,7 +35,8 @@ public function setUp()
             'name' => 'bobby testee',
             'email' => 'bobby@test.co.uk',
             'worker_centre' => $this->centre->id,
-            'alternative_centres.*' => $this->altCentres->pluck('id')->all()
+            'alternative_centres.*' => $this->altCentres->pluck('id')->all(),
+            'downloader' => 1,
```

Comment:
set here, `$this->data` will be used in both new and update tests.
In the `testICanStoreACentreUser()` we should see(), or seeInDatabase() that the `'downloader' => 1` has _not_ affected_ the default CentreUser construction of `0`. That way, we test that CentreUsers are created without that power.